draft-ppm-protocol.md

```diff
@@ -879,26 +915,23 @@ as follows:
 struct {
   TaskID task_id;
   Interval batch_interval;
-  select (protocol) { // Protocol corresponding to task_id
-    case prio: PrioCollectReq;
-    case hits: HitsCollectReq;
-  }
+  opaque output_param<0..2^16-1>;
 } CollectReq;
 ~~~
 
 The named parameters are:
 
 * `task_id`, the PPM task ID.
 * `batch_interval`, the request's batch interval.
-
-The remainder of the message is dedicated to the protocol-specific request
-parameters.
-
-Depending on the PPM scheme and how the leader is configured, the collect
-request may cause the leader to send a series of aggregate requests to the
-helpers in order to compute their share of the output. Alternately, the leader
-may already have made these requests and can respond immediately. In either case
-it responds to the collector's request as follows.
+* `output_param`, the output parameter with which the VDAF [VDAF] will be
+  evaluated.
+
+Depending on the VDAF and how the leader is configured, the collect request may
+cause the leader to send a series of aggregate requests to the helpers in order
+to compute their share of the output. Alternately, if `output_share` is empty or
```

Comment:
I think this only makes sense if changed to:
```suggestion
to compute their share of the output. Alternately, if `output_param` is empty or
```
Also, what should the collector put in `output_param` if it is "empty or a well-known value"? Empty byte sequence?